src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIKnight.java

```diff
@@ -9,7 +9,8 @@
 
 import java.util.List;
 
-import static com.minecolonies.coremod.entity.ai.util.AIState.*;
+import static com.minecolonies.coremod.entity.ai.util.AIState.DECIDE;
+import static com.minecolonies.coremod.entity.ai.util.AIState.GUARD_ATTACK_PHYSICAL;
 
 @SuppressWarnings("squid:MaximumInheritanceDepth")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:MaximumInheritanceDepth' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)